index.js

```diff
@@ -27,7 +43,10 @@ const checkAndRenameRepo = async () => {
         const stars = await getRepoStars()
 
         if (lastStars !== stars) {
-            await renameRepo(`This-Repo-Has-${stars}-Stars`)
+            await renameRepo({
+                name: `This-Repo-Has-${stars}-Stars`,
+                description: `Yes, it's true ${getMilestone(starts)}`
```

Comment:
You have misspelled `star` as `start`.

Reply:
@tianxiemaochiyu, don't worry about it, I'll fix it